votingpool/db.go

```diff
@@ -22,12 +22,13 @@ import (
 	"fmt"
 
 	"github.com/btcsuite/btcwallet/snacl"
-	"github.com/btcsuite/btcwallet/waddrmgr"
 	"github.com/btcsuite/btcwallet/walletdb"
 )
 
+// TODO: Rename votingPoolID to poolID everywhere in this file.
```

Comment:
Yeah, I refrained from doing so in this branch because it's not really related to the changes here. Happy to do so here if you won't mind the extra diff pollution when reviewing.